src/main/telemetry/hott.c

```diff
@@ -408,6 +481,10 @@ static inline void hottSendEAMResponse(void)
     hottSendResponse((uint8_t *)&hottEAMMessage, sizeof(hottEAMMessage));
 }
 
+static inline void hottSendTEXTMODEResponse(void)
```

Comment:
Don't use `inline`. The compiler optimisation will automatically inline where appropriate.